diffpy/pdfmorph/pdfplot.py

```diff
@@ -128,9 +133,6 @@ def comparePDFs(pairlist, labels=None, rmin = None, rmax = None, show = True,
     ymax = max(diff[sel])
     offset = -1.1*(ymax - ymin)
 
-    # Set up the plot
-    _configure()
```

Comment:
if this function is not being used any more, we should delete it (the function, not the call to it). Again, do it on a separate commit so we can revert if necessary.